<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UTableInfo</class>
 <widget class="QWidget" name="UTableInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>560</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>GraphWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
     <item>
      <widget class="QFrame" name="verticalFrame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutGraph">
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionSelectComponent">
   <property name="text">
    <string>Select Component</string>
   </property>
  </action>
  <action name="actionselectAllComponents">
   <property name="text">
    <string>Select All Components</string>
   </property>
  </action>
  <action name="actionDeleteAll">
   <property name="text">
    <string>Delete all Components</string>
   </property>
  </action>
  <action name="actionDeleteCurrentItem">
   <property name="text">
    <string>Delete current row</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
